### ISSUE TITLE
Possibility of a QML Example project?

### ISSUE THREAD

**Issue by Allstar12345** [mguludag/QStyleHelper#2]
Do you have a QML example project that replicates that in the video please?

I've gotten the Title bar to work correctly with  `QStyleHelper::setTitleBarDarkColor();` on an existing project
but not able to get the mica background in the window working correctly on that or a new test project, I assume there's some flags that need to be set on either side.

The project is great by the way thank you, my App looks so much better in dark mode without that horrible white Title Bar!

I wonder if there is a way to replicate the Mica effect in a QML component 🤔 it would look amazing since Qt aren't interested in their Apps looking good on Windows.

Cheers

**Comment by mguludag** [comment]
You have to simply set background color to `transparent`
This is the example QML file:
```QML
import QtQuick 2.12
import QtQuick.Controls 2.12
import QtQuick.Controls.Universal 2.12

import QtQml 2.12

ApplicationWindow {
    visible: true
    color: "transparent"
    width: 640
    height: 480

    Button{
        id: btn
        text: "sample button"
        anchors.centerIn: parent
    }
}
```

**Comment by Allstar12345** [comment]
Seems to be working a little odd for me, this is what I get with the code on the screen and your qml example:


![image](https://user-images.githubusercontent.com/8020981/229293265-27e67dd3-82f3-4592-8fb0-96d85d980ebe.png)



If I place QStyleHelper::setTitleBarDarkColor(); in the same place as the readme example I just get a white title bar and same transparent window, on Qt 5.15.2, Win 11 25330

**Comment by mguludag** [comment]
In `qstylehelper.hpp` line: 325 you try to change `build < 22523` to `build > 22523`. If this does not work, windows changed some values from its API on newer builds.

**Comment by Allstar12345** [comment]
Doesn't appear to make any difference:

![image](https://user-images.githubusercontent.com/8020981/229295515-92c8f35e-4b15-44bb-8715-1a3c0f53f2aa.png)
![image](https://user-images.githubusercontent.com/8020981/229295526-c9061b29-183e-47f7-befd-e48fb24f3e58.png)

Just to confirm everything looks correct on the main.cpp? 

Also is there any file or anything I can test for you on this windows build?

**Comment by mguludag** [comment]
> Doesn't appear to make any difference:
> 
> ![image](https://user-images.githubusercontent.com/8020981/229295515-92c8f35e-4b15-44bb-8715-1a3c0f53f2aa.png) ![image](https://user-images.githubusercontent.com/8020981/229295526-c9061b29-183e-47f7-befd-e48fb24f3e58.png)
> 
> Just to confirm everything looks correct on the main.cpp?

Yes, everything true on main.cpp. You are probably use windows insider builds. Works well on Win11 22H2. I don't know new parameters for insider builds.

**Comment by Allstar12345** [comment]
I am using Insider builds yes the Canary one, is there any file I can take from this build that may help you resolve the issue?

**Comment by mguludag** [comment]
I'll look at the ms docs and other sites for new parameters for mica on insider builds. Thank you for your help :)

**Comment by Allstar12345** [comment]
No problem :)

I'm not sure if this makes any difference but I just tried the Acrylic effect on Win 10 22H2 and that has the same effect, Black Title bar but clear Window with no blur, I wonder if something is wrong at my end.


Edit:     QStyleHelper::setAcrylicBlurWindow(QGuiApplication::allWindows(), true); is no change of behavior 



![image](https://user-images.githubusercontent.com/8020981/229297240-0bf05d8b-f6d4-4b77-9924-049ee9c402c4.png)
